tools/integration_utils/altdacommitment_parser/display.go

```diff
@@ -0,0 +1,250 @@
+package altdacommitment_parser
+
+import (
+	"encoding/hex"
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/Layr-Labs/eigenda/api/clients/v2/coretypes"
+	"github.com/Layr-Labs/eigenda/api/proxy/common/types/certs"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+// versionByteString returns a string representation of the version byte for display
+func versionByteString(v certs.VersionByte) string {
+	switch v {
+	case certs.V0VersionByte:
+		return "EigenDA V1"
+	case certs.V1VersionByte:
+		return "EigenDA V2 Legacy"
+	case certs.V2VersionByte:
+		return "EigenDA V2 with V3 Cert"
+	default:
+		return fmt.Sprintf("Unknown (0x%02x)", byte(v))
+	}
+}
+
+// DisplayPrefixInfo displays the parsed commitment structure information
+func DisplayPrefixInfo(parsed *PrefixMetadata) {
+	fmt.Printf("Decoded hex string to binary (%d bytes)\n", parsed.OriginalSize)
```

Comment:
super unnecessary and low priority but it'd be kinda cool to calculate the expected calldata costs as well here